src/image-sync-manager.ts

```diff
@@ -24,25 +23,29 @@ export class ImageSyncManager {
 
   private _syncEvents = () => {
     this._eventManager.listen(this._mainPlayer, this._mainPlayer.Event.TIMED_METADATA, this._onTimedMetadata);
-    this._eventManager.listen(this._mainPlayer, TIMED_METADATA_ADDED, this._onTimedMetadataAdded);
+    this._eventManager.listen(this._mainPlayer, this._mainPlayer.cuePointManager.TIMED_METADATA_ADDED, this._onTimedMetadataAdded);
   };
 
-  private _onTimedMetadata = ({payload}: {payload: {cues: Array<{track: {label: string; language: string}; value: {data: {id: string}}}>}}) => {
-    if (payload.cues[0]?.track?.label === 'KalturaCuePoints' && payload.cues[0]?.track?.language === SLIDE_CUE_POINT_TYPE) {
+  private _onTimedMetadata = ({payload}: {payload: {cues: Array<{track: {label: string}; value: {data: {id: string; cuePointType: string}}}>}}) => {
+    if (
+      payload.cues[0]?.track?.label === this._mainPlayer.cuePointManager.KalturaCuePointsTextTrack &&
+      payload.cues[0]?.value?.data?.cuePointType === ThumbCuePointType
+    ) {
       this._imagePlayer.setActive(payload.cues[0].value.data.id);
     }
   };
 
-  private _onTimedMetadataAdded = ({payload}: {payload: {cues: Array<{value: {key: string; type: string; data: Record<string, string>}}>}}) => {
+  private _onTimedMetadataAdded = ({payload}: {payload: {cues: Array<{value: {key: string; data: Record<string, string>}}>}}) => {
```

Comment:
can we make the `TIMED_METADATA_ADDED` payload same as `TIMED_METADATA` payload?